src/frontend/sider/Oppgavebenk/OppgavelisteNavigator.tsx

```diff
@@ -30,19 +20,16 @@ const OppgavelisteNavigator: React.FunctionComponent = () => {
     return (
         <HStack align="center" justify="space-between" wrap={false}>
             |
-            <StyledSpan>
+            <Box paddingInline="4">
```

Comment:
Samme her som over. Er det noen standard avstander i Aksel vi burde lene oss på fremfor å "hardkode" disse verdiene?

Reply:
Disse verdiene mapper om til [spacing tokens](https://aksel.nav.no/grunnleggende/styling/design-tokens#0cc9fb32f213) i Aksel. Så 4 mapper om til 1rem. Det jeg ser er at space-4 og 4 får samme resultat, jeg kan eventuelt gå over til å bruke "space-" prefikset, for å gjøre dette klarere 🙌

Reply:
Sjekket med aksel-teamet og tokener med `space-x` er den nye standarden. Jeg fikser!